File: app/src/main/java/com/bytedance/tiktok/activity/ShowImageActivity.java
package com.bytedance.tiktok.activity;

import android.widget.ImageView;
import com.bytedance.tiktok.R;
import com.bytedance.tiktok.base.BaseActivity;
import butterknife.BindView;

public class ShowImageActivity extends BaseActivity {
    @BindView(R.id.iv_head)
    ImageView ivHead;

    @Override
    protected int setLayoutId() {
        return R.layout.activity_show_image;
    }

    @Override
    protected void init() {
        ivHead.setOnClickListener(v -> finish());
    }
}

File: app/src/main/java/com/bytedance/tiktok/activity/ShowImageActivity.kt
package com.bytedance.tiktok.activity

import android.view.View
import com.bytedance.tiktok.base.BaseBindingActivity
import com.bytedance.tiktok.databinding.ActivityShowImageBinding

class ShowImageActivity : BaseBindingActivity<ActivityShowImageBinding>({ActivityShowImageBinding.inflate(it)}) {

    override fun init() {
        binding.ivHead.setOnClickListener { v: View? -> finish() }
        val headRes = intent.getIntExtra("res", 0)
        binding.ivHead.setImageResource(headRes)
    }
}